autogpt_platform/backend/backend/data/model.py

```diff
@@ -113,6 +113,7 @@ def SchemaField(
     advanced: Optional[bool] = None,
     secret: bool = False,
     exclude: bool = False,
+    hidden: Optional[bool] = None,
```

Comment:
can we default this to false?

Reply:
`advanced` is also `Optional[bool] = None`. This is so they don't get included in `json_extra` if not set.

I can change the check to `if v is not None and v is not False` and make them default `False`.

Reply:
Update: no I can't. It causes a bunch of failing tests because `Graph._generate_schema(..)` produces different output than `BlockSchema.jsonschema()`.